core/src/test/java/org/apache/calcite/rel/rules/DateRangeRulesTest.java

```diff
@@ -88,8 +88,7 @@
         "UTC",
         is("AND(AND(>=($8, 2014-01-01), <($8, 2015-01-01)),"
             + " AND(>=($8, 2014-06-01), <($8, 2014-07-01)))"),
-        is("AND(>=($8, 2014-01-01), <($8, 2015-01-01),"
-            + " >=($8, 2014-06-01), <($8, 2014-07-01))"));
+        is("SEARCH($8, Sarg[[2014-06-01\u20252014-07-01)])"));
   }
```

Comment:
Do we have to use `\u2025`? It seems less readable.

Reply:
I tried using unicode characters for infinity and "..". I got errors from checkstyle about line length, probably because the file encoding had been switched from ascii to unicode. It seemed simpler to revert to ascii. Most of our java files are ascii.

Reply:
I guess the error was Line is longer than [100 characters (found 103)](https://github.com/apache/calcite/runs/1046107924?check_suite_focus=true#step:4:382), wasn't it?
Then should we split the line like we usually do in that case?
Should we increase the limit to 120 or so chars?

I'm afraid `ascii` in the file makes the code way harder to read and reason about :((

Reply:
The line was nowhere near 100 characters. It was a checkstyle bug. Simplest thing is to keep the files ascii.

Reply:
I'm afraid the line could easily be `103 characters` as a single glyph might take multiple code points (Java chars) https://github.com/apache/calcite/commit/90663bb534e5c36cb3eb3964c90254491edfba0f#diff-4f1866bc93868c9718e968b312ba9422R301

So what do you think of wrapping the line (or increasing the limit)?

Reply:
I don't care about how many characters (or bytes) the line takes. I care about how many positions it takes in my text editor. I think checkstyle is checking the wrong thing, so it's a checkstyle bug.

The simplest thing is to revert to ascii and use \u escapes. Which is what I have done.

Reply:
Checkstyle developers agree. It [is a bug](https://github.com/checkstyle/checkstyle/issues/5089), and they fixed it in March. Still, simplest thing is to use `\u` escapes.

Reply:
>The simplest thing is to revert to ascii and use \u escapes

I would strongly disagree. The simplest thing (solution or workaround) is to wrap the line a bit earlier, so it is readable by humans.

`\u` escapes are not really readable, and it makes the code much harder to review and maintain.